packages/@react-spectrum/tooltip/docs/Tooltip.mdx

```diff
@@ -0,0 +1,195 @@
+
+```tsx example
+<TooltipTrigger delay={0}>
+  <ActionButton aria-label="Save"><Save /></ActionButton>
+  <Tooltip>Saving applies your new settings right away.</Tooltip>
+</TooltipTrigger>
+```
+
+### WarmUp / Cooldown
+
+Tooltips have a warm up and cool down period, [see documentation](https://spectrum.corp.adobe.com/page/tooltip/#Warmup-and-cooldown).
+Only one tooltip can be open at a time.
+```tsx example
+<Flex gap="size-200">
+  <TooltipTrigger>
+    <ActionButton>Hover me</ActionButton>
+    <Tooltip>I come up after a delay.</Tooltip>
+  </TooltipTrigger>
+  <TooltipTrigger>
+    <ActionButton>Then hover me</ActionButton>
+    <Tooltip>If you did it quickly, I appear immediately.</Tooltip>
+  </TooltipTrigger>
+</Flex>
+```
+
+## Tooltip placement
+
+Tooltips support a variety of placement options.
+
+### Placement
+
+The Tooltip's placement with respect to its trigger element can be adjusted using the `placement`
+prop. See the props table [below](#tooltiptrigger-props) for a full list of available placement combinations.
+
+```tsx example
+<TooltipTrigger placement="end">
+  <ActionButton aria-label="Foo">Placement</ActionButton>
+  <Tooltip>In left-to-right, this is on the right. In right-to-left, this is on the left.</Tooltip>
+</TooltipTrigger>
+```
+
+### Offset and cross offset
+
+The Tooltip's offset with respect to its trigger can be adjusted using the `offset` and
+`crossOffset` props. The `offset` prop controls the spacing applied along the main axis between the element and its
+trigger whereas the `crossOffset` prop handles the spacing applied along the cross axis.
+
+Below is a tooltip offset by an additional 50px above the trigger.
+```tsx example
+<TooltipTrigger offset={50}>
+  <ActionButton aria-label="Offset from trigger">Offset</ActionButton>
+  <Tooltip>This will shift up.</Tooltip>
+</TooltipTrigger>
+```
+Below is a tooltip cross offset by an additional 100px to the right of the trigger.
+```tsx example
+<TooltipTrigger crossOffset={100} placement="bottom">
+  <ActionButton aria-label="Cross Offset from trigger">Cross Offset</ActionButton>
+  <Tooltip>This will shift over to the right.</Tooltip>
+</TooltipTrigger>
+```
+
+## Events
+
+TooltipTriggers accepts an `onOpenChange` handler which is triggered whenever the Tooltip is shown or hidden.
+
+The example below uses `onOpenChange` to update a separate element with the current open state of the
+Dialog.
+
+```tsx example
+function Example() {
+  let [state, setState] = React.useState(false);
+
+  return (
+    <Flex alignItems="center" gap="size-100">
+      <TooltipTrigger onOpenChange={(isOpen) => setState(isOpen)}>
+        <ActionButton aria-label="Resize"><Resize /></ActionButton>
+        <Tooltip>Resize text.</Tooltip>
+      </TooltipTrigger>
+      <Text>Current open state: {state.toString()}</Text>
+    </Flex>
+  );
+}
+```
+
+## Props
+
+### TooltipTrigger Props
+<PropTable component={docs.exports.TooltipTrigger} links={docs.links} />
```

Comment:
Noticed some of the props have missing descriptions (TooltipTrigger's `isDisabled` and `delay`, Tooltip's `variant`, `placement`, etc)